_build/html/_sources/dev-manual/api/api-overview.rst.txt

```diff
@@ -41,7 +45,7 @@ Using the APIs
 
 All calls to the API endpoints require authentication with username and API key. These can be submitted as GET parameters (e.g. ``?username=demo&api_key=e6282adabed84e39ffe451f8bf6ff1a67c1fc9f2``) or in the  header (e.g. ``Authorization: ApiKey demo:e6282adabed84e39ffe451f8bf6ff1a67c1fc9f2``).
 
-There are a number of API endpoints in the Archivematica Dashboard, some of which are proxies to the Storage Service. For resource and endpoint descriptions, consult the :ref:`am-api-reference.rst`
+There are a number of API endpoints in the Archivematica Dashboard, some of which are proxies to the Storage Service. For resource and endpoint descriptions, consult the :ref:`am-api-reference.rst` section.
```

Comment:
I wonder if all folks accessing this documentation will know the difference between, say, a dashboard, and an Archivematica? We don't mention what the dashboard is and why it is important in the overview, so I may be tempted to avoid using dashboard at all. Alternatively, there may also be an opportunity to provide some richer information for the curious user, to describe the layout of Archivematica, i.e. a handful of django applications, where the dashboard is commonly recognized as its front-end component and provides API access as well as Archviematica's GUI?

I don't know how best to do that though. What do you think? We could perhaps rustle up a diagram too if we haven't anything suitable?